### DISCUSSION TITLE
Cloning and running rapidpro locally

### DISCUSSION THREAD

**Discussion by samueltuyizere** [nyaruka/rapidpro#5182] [Q&A] [answered]
I would love to explore the projects, but rapidpro/rapidpro repo doesn't have any instructions on how to set it up and run it locally. I tried ```yarn tw-watch``` but it hangs. Any help on this would be appreciated.

**Comment by norkans7** [comment, marked as answer]
I think you want to start from https://github.com/rapidpro/rapidpro?tab=readme-ov-file
or directly to http://rapidpro.github.io/rapidpro

**Reply by ericnewcomer** [reply]
Additionally, just to answer your question about `% yarn tw-watch` more directly, this is the tailwind watch command which isn't hanging -- it's monitoring the tailwind.scss file for changes and compiling if needed. This is used for development only.

**Comment by ronna** [comment]
Hi @samueltuyizere 
> I would love to explore the projects, but rapidpro/rapidpro repo doesn't have any instructions on how to set it up and run it locally. I tried `yarn tw-watch` but it hangs. Any help on this would be appreciated.

https://riseuplabs.com/rapidpro-installation-guide/

**Reply by samueltuyizere** [reply]
Thanks for sharing this @ronna , I will check it ut and revert.

**Comment by rowanseymour** [comment]
We recently put together a docker compose based on the latest 9.2 stable release which might be of some help https://github.com/nyaruka/rapidpro-docker
